src/setuptools_scm/_integration/deprecation.py

```diff
@@ -0,0 +1,20 @@
+import warnings
+
+from pathlib import Path
+
+
+def warn_dynamic_version(path: Path, section: str, expression: str) -> None:
+    warnings.warn(
+        f"{path}: at [{section}]\n"
+        f"{expression} forcing setuptools to override the version setuptools-scm sets\n"
```

Comment:
English isn't my first language, but I think there's a missing "is" before "forcing", and possibly a missing "that" before "setuptools-scm".